pipelines/rj_crm__relatorio_salesforce/tasks.py

```
@@ -0,0 +1,405 @@
+    flow com dump_mode=append duplicaria o mes.
+    """
+    project_id = PipelineConstants.BILLING_PROJECT_ID.value
+    client = bigquery.Client(
+        credentials=Base(bucket_name=project_id)._load_credentials(mode="prod"),
+        project=project_id,
+    )
+    query = f"""
+        SELECT COUNT(1) AS n
+        FROM `{project_id}.{dataset_id}.{table_id}`
+        WHERE data_particao BETWEEN DATE('{report_month}-01') AND LAST_DAY(DATE('{report_month}-01'))
+    """
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)